puma/tests/hlplots/test_results.py

```diff
@@ -72,27 +72,27 @@ def test_add_taggers_with_cuts(self):
         results.add_taggers_from_file(taggers, fname, cuts=cuts)
         self.assertEqual(list(results.taggers.values()), taggers)
 
-    def test_add_taggers_hbb(self):
-        # TODO: delete this when we bump the tools version
-        def structured_from_dict(d: dict[str, np.ndarray]) -> np.ndarray:
-            """Convert a dict to a structured array.
-
-            Parameters
-            ----------
-            d : dict
-                Input dict of numpy arrays
-
-            Returns
-            -------
-            np.ndarray
-                Structured array
-            """
-            from numpy.lib.recfunctions import unstructured_to_structured as u2s
-
-            arrays = np.column_stack(list(d.values()))
-            dtypes = np.dtype([(k, v.dtype) for k, v in d.items()])
-            return u2s(arrays, dtype=dtypes)
+    # TODO: delete this when we bump the tools version
+    def structured_from_dict(self, d: dict[str, np.ndarray]) -> np.ndarray:
```

Comment:
Since you are modifying this... you could bump the tools package version to 0.1.10 and use the function from there? https://github.com/umami-hep/atlas-ftag-tools/releases/tag/v0.1.10

Reply:
Sure, but we need to push this bugfix first
https://github.com/umami-hep/atlas-ftag-tools/pull/54